src/main/java/com/example/survey/models/Question.java

```diff
@@ -0,0 +1,24 @@
+package com.example.survey.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public abstract class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    protected long id;
+    protected String question;
+    public abstract String getQuestion();
+    public abstract void setQuestion(String q);
```

Comment:
My OOP is pretty rusty, why can't we implement these in the abstract class? Is it because abstract classes don't allow for implementations since they are abstract?

Reply:
nvm, I get it now